### DISCUSSION TITLE
Dynamisches Theaterstück

### DISCUSSION THREAD

**Discussion by TopperDEL** [Wendelpunkt/ToypadChallenge#28] [Ideas]
Ich könnte mir eine kleine Anwendung in Form eines bayrischen Theaterstücks vorstellen. Jede Figur stellt eine Figur auf der Bühne dar. Die Geschichte wird von ChatGPT geschrieben. Stellt man eine Figur auf das pad, betritt sie den Raum. Nimmt man sie weg geht sie (begründet) ab.
Die KI schreibt die Geschichte je nachdem, welche Personen gerade auf der Bühne stehen, hereinkommen oder wieder gehen. Keine Ahnung, ob das funktionieren würde - aber ich fände es witzig. :)

**Comment by tomwendel** [comment]
Hahaha! Wie geil ist denn bitte DIESE Idee?! Ich bin sehr gespannt! Mal sehen, ob man da noch eine Text-to-Speech Anbindung mit unterschiedlichen Stimmen bekommt.

**Reply by TopperDEL** [reply]
Sollte ja heute alles möglich sein. :)
